command/inspect.go

```diff
@@ -52,6 +50,50 @@ func (c *InspectCommand) Run(args []string) int {
 		return 1
 	}
 
+	// If args not specified but output format is specified, format and output the jobs data list
+	if len(args) == 0 {
+		var format string
```

Comment:
It should be possible to use the `-template` flag when specifying a particular job